src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/MessageValueProvider.cs

```diff
@@ -73,9 +73,10 @@ public string ToInvokeString()
             return Convert.ToBase64String(clonedMessage.Body);
         }
 
-        private static Task<string> GetBytesLengthAsync(Message clonedMessage)
+        private static Task<string> GetDescriptionAsync(Message clonedMessage)
         {
-            string description = string.Format(CultureInfo.InvariantCulture, "byte[{0}]", clonedMessage.Body.Length);
+            string description = clonedMessage.Body != null 
```

Comment:
Add null checks in : GetBase64StringAsync and GetTextAsync

Reply:
Added